src/realtime/realtime-note/realtime-note.repository.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * SPDX-FileCopyrightText: 2022 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import { Injectable } from '@nestjs/common';
+
+import { Note } from '../../notes/note.entity';
+import { RealtimeNote } from './realtime-note';
+
+@Injectable()
+export class RealtimeNoteRepository {
```

Comment:
Please document why this is a separate class and not part of `RealtimeNoteService` anymore

Reply:
Where?

Reply:
Reasons for the change should be documented in the commit message

Reply:
ok. done